### PR TITLE
Fix #1535: Use Lombok Getter and Setter in Entities

### LINKED ISSUE #1535: Use Lombok Getter and Setter in Entities

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationHistoryEntity.java [hunk 1]
```diff
@@ -35,172 +37,63 @@
...
+     * Record ID.
+     */
     @Id
     @SequenceGenerator(name = "pa_activation_history", sequenceName = "pa_activation_history_seq", allocationSize = 1)
     @GeneratedValue(strategy = GenerationType.AUTO, generator = "pa_activation_history")
     @Column(name = "id")
     private Long id;
 
+    /**
+     * Related activation.
```
Comment (banterCZ) [inline powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationHistoryEntity.java:56]: All JavaDocs are taken from getters.

File: powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationRecordEntity.java [hunk 2]
```diff
@@ -41,635 +43,206 @@
...
 
+    /**
+     * Activation flags.
+     */
     @Column(name = "flags")
     @Convert(converter = ActivationFlagConverter.class)
     private final List<String> flags = new ArrayList<>();
 
+    /**
+     * Base64 encoded server private key
```
Comment (romanstrobl) [inline powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationRecordEntity.java:129]: Inconsistent usage of periods at the end of sentence in this class.

**Review by romanstrobl — APPROVED** [review]
Looks good, just one minor typo.

File: powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationRecordEntity.java (hunk 2 above)
Comment (banterCZ) [reply powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationRecordEntity.java:129]:
```suggestion
     * Base64 encoded server private key.
```

**Review by romanstrobl — APPROVED** [review]
Approved.

**Pull request merged**